geemusic/intents/playback.py

```diff
@@ -81,6 +82,25 @@ def prev_song():
 
         return audio().play(stream_url)
 
+@ask.intent("AMAZON.ShuffleOnIntent")
+def shuffle_tracks():
+    shuffled_songs = queue.song_ids
+    
+    if shuffled_songs == [] or None:
+        return statement("There are no songs to shuffle.")
+    else:
+        api = GMusicWrapper.generate_api()
+    
+        # This will randomly reorder the song ids thereby "shuffling" the tracks
+        random.shuffle(shuffled_songs)
```

Comment:
actually if the `shuffle` operation is in the `MusicQueue` I think using `shuffle` is okay.

I have a bias against using any in-place operations, as I usually work with react/redux where mutable data types are a big nono. in this case it's fine though (and you're right, it's a bit more clear).